examples/subplots_axes_and_figures/figure_title.py

```diff
@@ -1,12 +1,17 @@
 """
-============
-Figure title
-============
+=============================================
+Figure labels: suptitle, supxlabel, supylabel
+=============================================
 
-Each subplot can have its own title (`.Axes.set_title`). Additionally,
-`.Figure.suptitle` adds a centered title at the top of the figure.
+Often it is desirable to have a title for a figure, while each subplot gets
+its own title using `.FigureBase.suptitle`.
```

Comment:
This is a somewhat misleading wording. Gives the impression that `suptitle` would create the indivicual subplot titles.

Reply:
Goops, yes, I'm confused what I thought I was saying here at all!